src/tools/ScenarioMeasurement/Startup/WindowsTraceSession.cs

```diff
@@ -104,7 +104,7 @@ private void InitWindowsKeywordMaps()
 
         public void EnableUserProvider(string provider)
         {
-            UserSession.EnableProvider(provider);
+            UserSession.EnableProvider(provider, TraceEventLevel.Verbose);
```

Comment:
Can verbosity be a parameter to `EnableUserProvider`? Not all providers are usable at the most verbose level. Some get very chatty.

Reply:
Good point!